_episodes/09-cmorization.md

```diff
@@ -115,7 +115,7 @@ run the existing one. There is a specific command available in the ESMValTool to
 run the CMORizer scripts:
 
 ```bash
-cmorize_obs -c <config-user.yml> -o <dataset-name>
+esmvaltool data format --config_file <config-user.yml>  <dataset-name>
```

Comment:
```suggestion
esmvaltool data format --config_file <path to config-user.yml>  <dataset-name>
```

Reply:
if the command is defined as `esmvaltool data format --config_file config-user.yml  <dataset-name>`, esmvaltool assumes that `config-user.yml` is in the current working directory and not the one in `~/.esmvaltool`, am I right?

Reply:
That is right. Since this episode requires to edit the `config-user.yml` a few times, I thought users would find it more intuitive to access the file from their working directory. But I agree with your suggestion that makes it less confusing.